### PR TITLE
Update typography

### PR DESCRIPTION
**Pull request by robinpokorny**

### CODE REVIEW

**Comment by robinpokorny** [comment]
When this is merged, screenshot should be updated before publishing.

File: app/app.js [hunk 1]
```diff
@@ -47,13 +47,20 @@
   const TodayLabel = ({ birthday }) => {
     const birthdayToday = isBirthdayToday(birthday)
     const text = birthdayToday
-      ? 'Okay, HB.'
-      : 'Today is NOT your birthday'
+      ? [
+        'Okay, ',
+        createElement('span', { className: 'label__emphasise'}, 'HB')
```
Comment (agudulin) [inline app/app.js]: dry a bit?

```js
const EmphasiseLabel = (text) => createElement('span', { className: 'label__emphasise'}, text)

const text = birthdayToday
  ? ['Okay, ', EmphasiseLabel('HB')]
  : ['Today is ', EmphasiseLabel('not'), ' your birthday']
```

File: css/style.css [hunk 2]
```diff
@@ -13,14 +15,17 @@ html, body, .root {
 }
 
 body, input {
-  font-family: -apple-system, "Helvetica Neue", "Lucida Grande", arial, sans-serif;
+  font-family: 'Unica One', sans-serif;
```
Comment (agudulin) [inline css/style.css]: not sure if I like this font family..

Comment (robinpokorny) [inline css/style.css]: I was looking for inspiration here https://femmebot.github.io/google-type/

Comment (agudulin) [inline css/style.css]: let's leave the system font here :)

File: app/app.js [hunk 3]
```diff
@@ -12,7 +12,7 @@
       try {
         return JSON.parse(window.localStorage.getItem(key))
       } catch (e) {
-        console.warning(`Can't get item from localStorage: ${key}`)
+        console.warn(`Can't get item from localStorage: ${key}`)
```
Comment (robinpokorny) [inline app/app.js]: https://developer.mozilla.org/en-US/docs/Web/API/Console/warn

File: app/app.js [hunk 4]
```diff
@@ -44,16 +44,19 @@
...
 
+  const EmphasiseLabel = (text) =>
+    createElement('span', { className: 'label__emphasise'}, text)
+
   const TodayLabel = ({ birthday }) => {
     const birthdayToday = isBirthdayToday(birthday)
     const text = birthdayToday
-      ? 'Okay, HB.'
-      : 'Today is NOT your birthday'
+      ? ['Okay, ', EmphasiseLabel('HB')]
```
Comment (agudulin) [inline app/app.js]: so we don't need `EmphasiseLabel` anymore :)

**Pull request merged**